dockerfiles/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM openjdk:8
+ARG HIVE_VERSION=3.0.0.13.3
+ARG HADOOP_VERSION_EE=3.2.0.12-EE-SNAPSHOT
+ARG PROMETHEUS_EXPORTER_VERSION="0.12.0"
+ARG USER="hive"
+ARG USERID="1515"
```

Comment:
from looking at the chef cookbooks, this is not the user_id for hive - it should be `1516` 
that seems to be the user for certs 
https://github.com/logicalclocks/kagent-chef/blob/master/attributes/default.rb#L10C1-L13C60